### ISSUE TITLE
[bug]: 

### ISSUE THREAD

**Issue by srt-jay** [invoke-ai/InvokeAI#3549]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### What version did you experience this issue on?

2.3.5

### What happened?

When installing on windows using the provided .bat file, it keeps stopping out of nowhere. I keep getting this error on each attempt:

INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https://pip.pypa.io/warnings/backtracking for guidance. If you want to abort this run, press Ctrl + C.

I don't know why it is halting so much, but it never finishes even if I leave the computer on for days. This is attempt 5 but it feels like 500....
What can I do to make this finish installing??? I'm about ready to tear my hair out. Can I make it use only relevant versions instead of checking literally every single one exhaustively? If so, can I do that without having to edit a file that is bigger than my curriculum vitae?
I did enable long path support, even tried running the .bat file with explicit admin privileges, and I ate a taco while waiting once hoping it would bring good luck. I think my computer wants to drop kick me across the room and idk if we can make-up this time.

### Screenshots

![installer_no_like_me](https://github.com/invoke-ai/InvokeAI/assets/136917794/72ebb33b-d4ad-42b0-99b1-27e939aebd5d)

### Additional context

Probably happens because, 1) everything I touch dies, 2) I don't know jack about python (running version 3.10 FYI) and 3) my tummy hurts :C

**Comment by srt-jay** [comment]
Also while looking at other support tickets, I noticed a trend of "this was an oversight, we'll work on a solution," as opposed to what I normally see in open-source which is "uhhhrrrr durrr user no know to code to fix lmao user so stupid, a 5 year old could figure it out" and I appreciate your team handling these issues better than a grand majority of open source devs do.

**Comment by srt-jay** [comment]
Seems to be a problem with Python itself. I fixed the issue, not entirely sure how, but I had to uninstall EVERYTHING related to python and I reinstalled the recommended version. However, I am unable to maintain other versions of python at the moment.

Problem: On windows with all CUDA deps met, the installer .bat file stops somewhere before config prompt, and seeks out too many versions of dependencies/libraries, and freezes (with task manager showing signs of memory error.) Installer will not continue, and remain stuck. Likely as a result of conflicting python versions.

Solution: Uninstall all python versions, RESTART, re-install python (only recommended version, no others) and re-install InvokeAI and everything works as planned.

Thanks for taking the time to make this software, it will be interesting to train it and see where things go.
